src/components/templates/settingsLayout.tsx

```diff
@@ -7,13 +7,13 @@ import { LogoCatLarge, PageTitle } from "@components/atoms";
 import { Sidebar, TitleTopbar } from "@components/organisms";
 import { SettingsMenu } from "@components/organisms/settings";
 
-export const SettingsLayout = () => {
+export const SettingsLayout = ({ variant }: { variant: "userSettings" | "organization" }) => {
```

Comment:
Here as well, I think we can pass the menuItems as an array instead of a variant.
It will make the SettingsMenu more modular, and not limited to only 2 variants, but for any menu with menu-items.

Reply:
discussed